relays/bin-substrate/src/cli/relay_headers_and_messages/chain_to_chain.rs

```diff
@@ -0,0 +1,211 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
```

Comment:
Probably rename this file to `relay_to_relay,rs` (and all nested structs), since both parachain and relay chains are *chains*, so it isn't clear from the name - what this file is about :)

Reply:
Done